pages/vi/vi-github-and-markdown.md

```diff
@@ -66,7 +66,7 @@ After you fork the repository, you will be on your page: &lt;username&gt;/open-l
 
 After you rename it, make sure to post a link to the github.io website you've just created (&lt;username&gt;.github.io) in the [gitter.im chat](https://gitter.im/open-learning-exchange/chat).
 
-![Example of github.io page](images/vi-example-githubio.png)
+![Example of github.io page](images/vi-example-githubio-chat.jpg)
```

Comment:
A better way to do this would be replacing the image, not just changing the name.

Also, it looks like you forgot to `git add` new screenshots?

Aside from that, if we follow `vi-github-and-markdown.md` from top to down, the personal profile .md file shouldn't exist at this point. Probably just use `> @dogi look at https://<username>.github.io ` instead of a screenshot here for simplicity?

Reply:
For the images, I uploaded them in the pages/vi/images. And I will remove the personal profile.md and will replace it. But for images I thought uploading them to the images and referencing them is enough but i will do the git add of new screenshots anyway.

Reply:
@lorugant Not sure what others think here. I feel for simplicity, it could just be:

> @dogi look at `https://<username>.github.io`

instead of a screenshot.

---

Regarding image, I'm not sure if I understand you correctly, where did you "uploaded them in the pages/vi/images"? If you look at http://rawgit.com/lorugant/lorugant.github.io/LBranch/index.html#!pages/vi/vi-github-and-markdown.md, the image is not there.

Reply:
Ok I understand the problem now. Thank you. I will fix it.